bdk-ffi/src/lib.rs

```diff
@@ -1749,4 +1755,33 @@ mod test {
             bdk::Error::Descriptor(Key(InvalidNetwork))
         )
     }
+
+    #[test]
+    fn test_blockchain_fee_estimator_electrum() {
+        let blockchain = Blockchain::new(BlockchainConfig::Electrum {
```

Comment:
Thanks for adding this PR, the code looks good. 

But the end to end test here hits an external server which should be avoided in a unit test. Since all that's left is some easy to review code I'd remove the test. 

If you want to create a separate PR to add a local regtest integration test in `bdk` for the `estimate_fee` function it would go here:
https://github.com/bitcoindevkit/bdk/blob/master/src/testutils/blockchain_tests.rs

Also you should update the api-doc/kotlin docs, this currently needs to be done by hand. And you can add some example code there to for how to call/use your new function.

Reply:
OK, sounds good!  In https://github.com/bitcoindevkit/bdk-ffi/commit/f01e0e30f3dea71af192885a0e7c4691afd81c63 I have —
- removed the test
- added documentation on this api
- added a sample to the blockchain sample code (that is where this new api resides)